hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/snapshot/SnapshotDiffHandler.java

```diff
@@ -54,6 +54,11 @@ public class SnapshotDiffHandler extends Handler {
           " (optional)")
   private int pageSize;
 
+  @CommandLine.Option(names = {"-fd", "--use-full-diff"},
+      description = "perform full diff of snapshot without using" +
+          " optimised DAG based pruning approach (optional)")
+  private boolean fullDiff;
```

Comment:
This option would be useful for debugging. But for now let's hide this option by default to regular users (still usable). We can decide whether to unhide this later.

```suggestion
      description = "perform full diff of snapshot without using" +
          " optimised DAG based pruning approach (optional)",
      hidden = true)
  private boolean fullDiff;
```